releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -6,6 +6,15 @@
 
 [Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 2.5.0 - March 29, 2022
+
+* Add a refresh interval `spring.cloud.azure.appconfiguration.refresh-interval= 1d` that can be used to periodically reload all the key-value pairs in a configuration store. This can also be used to force update Key Vault References.
```

Comment:
Do we have a default value of one day? We shouldn't. It will make this a breaking change.

Reply:
This value isn't set by default. I just was using it as an example.

Reply:
It's not clear it's an example. Can you add what you said in the release notes and documentation (if not yet).

Reply:
It is clear in the documentation as it is in a table that says default value is null. https://microsoft.github.io/spring-cloud-azure/docs/azure-app-configuration/2.5.0/reference/html/index.html#forced-client-refresh

Reply:
I will just remove setting it as an example as it works like the other refresh intervals. i.e. taking a `Duration`.

Reply:
Also, made a new PR as this one was merged already. https://github.com/Azure/AppConfiguration/pull/627